app/src/main/res/layout/nav_section_item.xml

```diff
@@ -1,28 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@android:color/transparent"
-    android:orientation="vertical"
     android:importantForAccessibility="no"
+    android:orientation="vertical"
     android:paddingTop="8dp"
     android:paddingBottom="8dp">
 
     <View
         android:layout_width="match_parent"
         android:layout_height="1dp"
         android:background="?android:attr/listDivider"
-        tools:background="@android:color/holo_red_dark"/>
+        tools:background="@android:color/holo_red_dark" />
 
-    <TextView
+    <LinearLayout
         android:id="@+id/nav_feeds_filtered_message"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:gravity="start"
-        android:paddingStart="@dimen/listitem_icon_leftpadding"
-        android:paddingTop="4dp"
-        android:paddingEnd="@dimen/listitem_icon_leftpadding"
-        android:textColor="?android:attr/textColorSecondary"
-        android:textSize="@dimen/text_size_small" />
+        android:importantForAccessibility="no"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:layout_width="15dp"
+            android:layout_height="15dp"
+            android:layout_marginLeft="18dp"
+            android:layout_marginTop="6dp"
```

Comment:
These numbers are quite unusual. Material design usually uses numbers that are multiples of 8 (sometimes multiples of 4). I would use `marginStart` instead of `marginLeft` for right-to-left language support.

Reply:
To be honest, looking at it closely, the current placement also is a bit odd:
![Imagepipe_0.jpg](https://github.com/AntennaPod/AntennaPod/assets/11229646/49336c06-25dd-4868-83c7-c13a1822dc19)

Can we take the same start-margin as podcasts, such that the notification icon starts at the same indent?

Reply:
I think it's fine as it is, but aligning would be fine as well. @caoilte let me know which version you prefer, then we can merge it :)

Reply:
I don't have a strong opinion about the look and feel. I'm not familiar with the UX for subscriptions filtering and don't even know how to reproduce the podcast icons in your image or what it means. 

Progression as i understand it,
1) I tried to match the previous UI using whatever sizes and offsets made it look the same
2) you said I should use multipliers of 8 or 4 and I resized a number of things to make that work. As a result the icon got bigger. I couldn't make it look good using multiples of 4 at the previous size. 
3) now you're not sure if it should even match the previous UI.

I'm completely fine with whichever direction we go in I'm just not confident about what the next step should be.
- We can make the icon smaller again but I might need help doing it using combinations of 4 and 8 for margin etc
- I'd be happy to use the same start-margin as the podcasts and apologies that I'm such a bad eye for judging good UX.


Either way, if you want to tweak any of the settings to get it mergeable go right ahead. I'm always happy to have that sort of collaboration instead of going back and forth. Or I can go ahead with whichevr direction you think would be best. Thanks again for helping out so much. This has been fun.